### PR TITLE
Elhalper/15626/editor page test

### PR DESCRIPTION
**Pull request by elizabethhalper**
Creating three tests for the EditorPage:

- Sets project state from Redux store
- Loads project assets when state changes
- Raises onAssetSelected handler when an asset is selected from the sidebar

### CODE REVIEW

File: src/react/components/pages/editorPage/editorPage.test.tsx [hunk 1]
```diff
@@ -0,0 +1,127 @@
+import React from "react";
```
[the remark this answers is not in the document]
Comment (elizabethhalper) [reply src/react/components/pages/editorPage/editorPage.test.tsx]: done :)

File: src/react/components/pages/editorPage/editorPage.test.tsx [hunk 2]
```diff
@@ -0,0 +1,127 @@
...
+        const loadProjectSpy = jest.spyOn(props.actions, "loadProject");
+
+        const wrapper = createCompoent(store, props);
+        const editorPage = wrapper.find(EditorPage).childAt(0);
+
+        expect(loadProjectSpy).not.toBeCalled();
+        expect(editorPage.prop("project")).toEqual(testProject);
+    });
+
+    it("Loads project assets when state changes", () => {
```
[the remark this answers is not in the document]
Comment (elizabethhalper) [reply src/react/components/pages/editorPage/editorPage.test.tsx]: Wouldn't this just be an empty IAsset list? Or should I mock up a test asset and assert that it is added to the assets state param?

[the remark this answers is not in the document]
Comment (wbreza) [reply src/react/components/pages/editorPage/editorPage.test.tsx]: At this point if your mock project has assets - they should get loaded into the sidebar and then be state to the `assets` state param.  It wouldn't be empty, but rather equal to the length of the project assets.

File: src/react/components/pages/editorPage/editorPage.test.tsx [hunk 3]
```diff
@@ -0,0 +1,151 @@
...
+        const store = createStore(testProject, true);
+        const props = createProps(testProject.id);
+
+        const wrapper = createCompoent(store, props);
+        const editorPage = wrapper.find(EditorPage).childAt(0);
+
+        const editorSideBar = wrapper.find(EditorSideBar);
+        const onAssetSelectedSpy = jest.spyOn(editorSideBar.props(), "onAssetSelected");
+
+        // THOUGHT PROCESS: probs have to create jest.fn() => something for onAssetSelected
```
Comment (wbreza) [inline src/react/components/pages/editorPage/editorPage.test.tsx]: The sidebar component exposes a public `selectAsset` method where you can tell it to select a given asset... this should then raise the event handler that is bound in the editor page... I will make sure this is working as expected.

**Comment by wbreza** [comment]
@elizabethhalper - looks like there are a couple minor linting issues blocking the tests.

**Pull request merged**